src/lib/seam/components/AccessCodeTable/AccessCodeHealthBar.tsx

```diff
@@ -25,7 +28,9 @@ export function AccessCodeHealthBar({
   const warningAccessCodes = accessCodes.filter(
     (accessCodes) => accessCodes.warnings.length > 0
   )
-  const codesWithIssues = _.merge(erroredAccessCodes, warningAccessCodes)
+  const codesWithIssues = filterOutDuplicates(
+    erroredAccessCodes.concat(warningAccessCodes)
+  )
   const issueCount = codesWithIssues.length
```

Comment:
I think this would work and is more clear. We don't need the codes, we just need the count.
```suggestion
  const issueCount = accessCodes.filter(({ errors, warnings }) => errors.length > 0 || warnings.length > 0).length
```

Reply:
haha duh, you're right!

Reply:
thx for the clean up~